OpenEdX/Environment.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Environment.swift
```

Comment:
@saeedbashir  was this merged with latest develop? I believe this class was got rid in Config PR. Thank you

Reply:
I'm not able to find this class in my branch. It might be showing off because of merge.

Reply:
It's strange because I see you merged 'develop' which has this file deleted. Did you do git merge or rebase? Thank you

Reply:
I've rebased it. Sometimes I use the github web interface to resolve conflicts, and I guess that uses merge instead of rebase.